### DISCUSSION TITLE
super-charging-git-rebase-with-git-absorb/

### DISCUSSION THREAD

**Discussion by giscus** [andrewlock/blog-comments#135] [Announcements]
# super-charging-git-rebase-with-git-absorb/

In this post I describe the git-absorb tool, and show how it can super-charge your git rebase workflow, removing a huge amount of manual work!

https://andrewlock.net/super-charging-git-rebase-with-git-absorb/

**Comment by wildmichael** [comment]
You definitely should also take a look at [stacked-git](https://stacked-git.github.io/). No need to create fixup commits. Directly refresh, reorder, push and pop commits in your patch queue.
